app/assets/stylesheets/sufia/blacklight_gallery.scss

```diff
@@ -0,0 +1,12 @@
+/* These styles are very specific to override the thumbnails from blacklight gallery */
+
+#documents .grid .document .thumbnail {
+  padding: 0;
+}
+#documents .grid .document .thumbnail > a > img {
```

Comment:
These will be tough because they are very specific and coming from Blacklight. I matched the declaration to the letter in order to override. Might need to do `html > body` instead of the ID selector. Big hammer.